pages/linux/add-apt-repository.md

```diff
@@ -0,0 +1,19 @@
+# add-apt-repository
+
+> Manages apt repository definitions.
+
+- Add a new apt repository:
+
+`add-add-repository {{repository_spec}}`
+
+- Remove an apt repository:
+
+`add-apt-repository --remove {{repository_spec}}`
+
+- Update the package cache after adding a repository:
+
+`add-apt-repository --update {{repository_spec}}`
+
+- Also enable source packages:
```

Comment:
Maybe just `Enable source packages:` to match the format of the other command descriptions? 🤔